ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/boolean/activity-boolean-checkbox-answer/activity-boolean-checkbox-answer.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ActivityBooleanQuestionBlock } from '../../../../../models/activity/activityBooleanQuestionBlock';
+
+@Component({
+    selector: 'ddp-activity-boolean-checkbox-answer',
+    template: `
+        {{ block.answer }}
```

Comment:
Do we need it here ? 
it is just **true** / **false**.

Reply:
Nope, my bad, I forgot yo delete